prisma/migrations/20231115062915_add_ticker_whitelist/migration.sql

```diff
@@ -0,0 +1,13 @@
+-- CreateTable
+CREATE TABLE "TickerWhitelist" (
+    "id" SERIAL NOT NULL,
+    "ticker" TEXT NOT NULL,
+
+    CONSTRAINT "TickerWhitelist_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "TickerWhitelist_ticker_key" ON "TickerWhitelist"("ticker");
+
+INSERT INTO "TickerWhitelist"(ticker) VALUES
+('NCG'), ('FAV_CRYSTAL'), ('Item_NT_400000'), ('Item_NT_500000'), ('Item_T_40100015'), ('Item_T_40100016'), ('Item_T_40100017'), ('Item_T_10130002');
```

Comment:
`FAV` (Wrapping FAV token) uses `__` as a separator. (See https://github.com/planetarium/lib9c/pull/2226/files#diff-f8baba9adeba0192ee8619bbab5f7ed3de3ddd48cdfb48cfa5b28bf4527b8fc1R152-R155)

```suggestion
('NCG'), ('FAV__CRYSTAL'), ('Item_NT_400000'), ('Item_NT_500000'), ('Item_T_40100015'), ('Item_T_40100016'), ('Item_T_40100017'), ('Item_T_10130002');
```